frontend/packages/core/src/popper.tsx

```diff
@@ -89,7 +89,7 @@ export interface PopperProps
   extends Pick<ClickAwayListenerProps, "onClickAway">,
     Pick<MuiPopperProps, "placement"> {
   open: boolean;
-  anchorRef?: React.MutableRefObject<HTMLElement> | null;
+  anchorRef?: React.MutableRefObject<HTMLElement | null> | null;
```

Comment:
wdyt about removing this now since the PR was the correct fix?
```suggestion
  anchorRef?: React.MutableRefObject<HTMLElement | null>;
```

Reply:
ok. also how about we remove the question mark as well since we don't need the `undefined` type

Reply:
in looking at material ui popper docs, the anchorRef is an optional prop so I think it makes sense to keep the `?`

Reply:
got it